docs/secrets.md

```diff
@@ -73,6 +73,10 @@ of very large secrets which would exhaust apiserver and kubelet memory.
 However, creation of many smaller secrets could also exhaust memory.  More
 comprehensive limits on memory usage due to secrets is a planned feature.
 
+Pods must be created via the API server to use secrets.  (Kubelets support
```

Comment:
This sort of seems to imply that pods created by a replication controller can't use secrets. Someone who knows the system will know that replication controllers go through the apiserver, but it may confuse beginners.

I can't think of a better way to word it that doesn't make it sound even more confusing... It's fine how it is if you can't come up with an obvious better way to say it.